presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlanner.java

```diff
@@ -485,12 +485,16 @@ protected RelationPlan visitValues(Values node, Void context)
                 for (int i = 0; i < items.size(); i++) {
                     Expression expression = items.get(i);
                     expression = ExpressionTreeRewriter.rewriteWith(new ParameterRewriter(analysis.getParameters(), analysis), expression);
+                    // RelationPlanner should not invoke evaluateConstantExpression, which in turn invokes ExpressionInterpreter.
```

Comment:
make this `TODO` (here & below)

Reply:
it should go in separate commit

Reply:
This is closely related to the comments added in `ExpressionInterpreter.java`. I believe it's reasonable to be put together.